bootstrap-salt.sh

```diff
@@ -2474,7 +2496,7 @@ install_debian_7_deps() {
     fi
 
     # shellcheck disable=SC2086
-    wget $_WGET_ARGS -q http://debian.saltstack.com/debian-salt-team-joehealy.gpg.key -O - | apt-key add - || return 1
+    __fetch_verify http://debian.saltstack.com/debian-salt-team-joehealy.gpg.key 267d1f152d0cc94b23eb4c6993ba3d67 3100 | apt-key add - || return 1
```

Comment:
As for Debian 7, http://debian.saltstack.com/ should be considered as deprecated. See saltstack/salt#31816.
SaltStack team is finally providing up-to-date 2015.8 builds for Wheezy from their Corporate Repository: https://repo.saltstack.com/apt/debian/7/amd64/latest/.

So I guess the entire function needs to be rewritten.